index.html

```diff
@@ -131,7 +131,7 @@ <h2>Project name goes here</h2>
           </div>
 
           <div class="project-container project-5">
-            <img src="images/project-5.jpg" alt="project-5 image">
+            <img src="images/project-5.jpg" alt="macbook pro 16 standing on its corner">
 
             <div class="single-project-info">
               <h2>Project name goes here</h2>
```

Comment:
- [ ] Please change this to `h3` instead of `h2` as the Heading levels should have a meaningful hierarchy.